flask_resty/view.py

```diff
@@ -55,6 +98,23 @@ def set_item_response_meta(self, item):
         pass
 
     def make_response(self, data, *args, **kwargs):
+        """Create an HTTP response with the provided `data` in the response
+        body along with any metadata from the Flask-RESTy context.
```

Comment:
What is the "Flask-RESTy context"?

Reply:
See http://flask.pocoo.org/docs/1.0/appcontext/, similar to a request context but for state local to flask resty.

Reply:
Is it clear how one sets that metadata?

Reply:
Whoops looks like I missed the `meta` module